apps/playwright/tests/admin/process.spec.ts

```diff
@@ -0,0 +1,145 @@
+import { Pool } from "@gc-digital-talent/graphql";
+
+import { test, expect } from "~/fixtures";
+import PoolPage from "~/fixtures/PoolPage";
+import TeamPage from "~/fixtures/TeamPage";
+import { getDCM } from "~/utils/teams";
+import { getClassifications } from "~/utils/classification";
+import { loginBySub } from "~/utils/auth";
+
+test.describe("Process", () => {
+  const uniqueTestId = Date.now().valueOf();
+  const unassociatedSub = `playwright.sub.unassociated.${uniqueTestId}`;
+  const unassociatedPoolManagerEmail = `${unassociatedSub}@example.org`;
+  const associatedSub = `playwright.sub.associated.${uniqueTestId}`;
+  const associatedPoolManagerEmail = `${associatedSub}@example.org`;
+  const poolName = `pool auth test ${uniqueTestId}`;
+
+  let pool: Pool;
+
+  test.beforeAll(async ({ adminPage }) => {
+    const poolPage = new PoolPage(adminPage.page);
+    const teamPage = new TeamPage(adminPage.page);
+
+    const orphanTeam = await teamPage.createTeam({
+      name: `orphan-team.${uniqueTestId}`,
+    });
+
+    const unassociatedPoolManager = await adminPage.createUser({
+      email: unassociatedPoolManagerEmail,
+      sub: unassociatedSub,
+    });
+
+    await adminPage.addRolesToUser(unassociatedPoolManager.id, [
+      "guest",
+      "base_user",
+      "applicant",
+    ]);
+
+    await adminPage.addRolesToUser(
+      unassociatedPoolManager.id,
+      ["pool_manager"],
+      orphanTeam.id,
+    );
+
+    const team = await getDCM();
+
+    const associatedPoolManager = await adminPage.createUser({
+      email: associatedPoolManagerEmail,
+      sub: associatedSub,
+    });
+
+    await adminPage.addRolesToUser(associatedPoolManager.id, [
+      "guest",
+      "base_user",
+      "applicant",
+    ]);
+
+    await adminPage.addRolesToUser(
+      associatedPoolManager.id,
+      ["pool_operator"],
+      team.id,
+    );
```

Comment:
No because they do different things. The first one applies global roles where as the second one applies the roles within the scope of the team.

Reply:
Will it cause an error ?  If so, I feel like we should create another method to add roles within the scope of a team (e.g. addTeamRoleToUser) to prevent devs from running into an issue in the future.